src/interface/storage.thrift

```diff
@@ -521,6 +520,8 @@ struct LookupIndexRequest {
     // The list of property names. Should not be empty.
     // Support kVid and kTag for vertex, kSrc, kType, kRank and kDst for edge.
     4: optional list<binary>                return_columns,
+    // max row count of each partition in this response, 0 means no limit.
+    5: i64                                  limit = 0,
```

Comment:
Try `optional` and `NULL` means no limit instead zero.

Reply:
Or you could try max of i64

Reply:
Or negative value for this.

Reply:
I don't see any advantage in your approach. your approach may make the code more complex.

Reply:
The limit 0 is allowed.

Reply:
And the other limit field in request is optional too.

Reply:
> And the other limit field in request is optional too.

good point.